packages/odyssey-react/src/components/Icon/Icon.tsx

```diff
@@ -0,0 +1,102 @@
+/*!
+ * Copyright (c) 2021-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import { Children, isValidElement, cloneElement, useMemo, CSSProperties, ComponentPropsWithRef, forwardRef } from "react";
+import type { ReactNode } from "react";
+import { nanoid } from 'nanoid';
+
+import styles from "./Icon.module.scss";
+
+export type Props = {
+  /**
+   * Title text used by screen readers
+   * @default the name of the icon
+   */
+  title?: string;
```

Comment:
this should be required yea? icons won't be accessible to non-sighted users unless this is present

Reply:
Actually now that I think about this its a bit more nuanced - some icons will necessarily be for sighted users only in which case the rendered children should be `role="presentation"`. If the `title` prop is present that role should probably be something like `role="img"`. Lets adjust here